binr/rahash2/rahash2.c

```diff
@@ -23,9 +24,9 @@ void compare_hashes(const RHash *ctx, const ut8 *compare, int length, int *ret)
 	if (compare) {
 		// algobit has only 1 bit set
 		if (!memcmp (ctx->digest, compare, length)) {
-			printf ("rahash2: Computed hash matches the expected one.\n");
+			printf ("\nrahash2: Computed hash matches the expected one.\n");
```

Comment:
why this newline

Reply:
Because now, without `-l` there won't be a new line:

```
// No new line
Megabbets-PC$ rahash2 -a crc32 -s test
0x00000000-0x00000003 crc32: d87f7e0cMegabbets-PC$

// with new line
Megabbets-PC$ rahash2 -l -a crc32 -s test
0x00000000-0x00000003 crc32: d87f7e0c

// \n is required in order to print "Computed hash matches the expected one." in a new line
Megabbets-PC$ rahash2 -a crc32 -s test -c d87f7e0c
0x00000000-0x00000003 crc32: d87f7e0c
rahash2: Computed hash matches the expected one.
```

On master you have new line after the result of the hash, now you won't (without `-l`)